statistics/ExecutionSummaryCliParser.go

```diff
@@ -0,0 +1,106 @@
+package statistics
+
+import (
+	"regexp"
+	"strconv"
+)
+
+//ExecutionSummaryCliParser ...
+type ExecutionSummaryCliParser struct {
+}
+
+//Parse ...
+func (instance ExecutionSummaryCliParser) Parse(data string) ExecutionSummary {
+	return ExecutionSummary{
+		RunningTime:   parseRunningTime(data),
+		Throughput:    parseThroughput(data),
+		TotalRequests: parseTotalRequests(data),
+		TotalErrors:   parseTotalErrors(data),
+		Availability:  parseAvailability(data),
+		Bytes: ByteSummary{
+			TotalSent:     parseTotalBytesSent(data),
+			TotalReceived: parseTotalBytesReceived(data),
+		},
+		MeanResponseTime: parseMeanResponseTime(data),
+		MinResponseTime:  parseMinResponseTime(data),
+		MaxResponseTime:  parseMaxResponseTime(data),
+	}
+}
+
+func checkMatchString(match []string) string {
+	if len(match) != 2 {
```

Comment:
The result of the match is: 

1[0]. The full string
2[1]. The 1st matching group
n[n-1] The nth matching group

I agree it is a matching number.  I will assign it to a variable.